README.md

```diff
@@ -21,7 +23,8 @@ Add this to your `.pre-commit-config.yaml`:
 
 ### classification of imports
 
-`isort` isn't always the best at classifying imports.
+`isort` isn't always the best at classifying imports (although recent
+versions are much better).
```

Comment:
citation needed?

Reply:
yeah this hasn't improved at all:

```console
$ pre-commit  run --all-files
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing t.py

```
```diff
$ git diff
diff --git a/t.py b/t.py
index 1920b6b..2bbf88d 100644
--- a/t.py
+++ b/t.py
@@ -1,5 +1,4 @@
 import os
 
 import astpretty
-
 import t
```

please rebase out this part of the patch, thanks

Reply:
Sorry, this is obviously not actually true, I didn't fully page in the context for this issue before I started working on it :disappointed: